src/denoise.c

```diff
@@ -277,6 +277,7 @@ DenoiseState *rnnoise_create(RNNModel *model) {
 }
 
 void rnnoise_destroy(DenoiseState *st) {
+  opus_fft_free(common.kfft, 0);
```

Comment:
Not sure I understand the surrounding code correctly but I think this is only allocated on demand in `check_init` so it should only be deallocated if it has been initialized.

Reply:
fair enough. committed change to only free if common.init == 1